build.sbt

```diff
@@ -1202,18 +1158,20 @@ ThisBuild / scriptedPrescripted := { _ =>
 
 def otherRootSettings =
   Seq(
-    scripted := scriptedTask.evaluated,
-    scriptedUnpublished := scriptedUnpublishedTask.evaluated,
+    scripted := scriptedTask(false).evaluated,
+    scriptedUnpublished := scriptedTask(false).evaluated,
     scriptedSource := (sourceDirectory in sbtProj).value / "sbt-test",
     watchTriggers in scripted += scriptedSource.value.toGlob / **,
+    watchTriggers in scriptedUnpublished := (watchTriggers in scripted).value,
     scriptedLaunchOpts := List("-Xmx1500M", "-Xms512M", "-server") :::
       (sys.props.get("sbt.ivy.home") match {
         case Some(home) => List(s"-Dsbt.ivy.home=$home")
         case _          => Nil
       }),
-    publishAll := { val _ = (publishLocal).all(ScopeFilter(inAnyProject)).value },
-    publishLocalBinAll := { val _ = (publishLocalBin).all(ScopeFilter(inAnyProject)).value },
-    aggregate in bintrayRelease := false
+    publishLocalBinAll := Def.taskDyn {
+      (Compile / (if (insideCI.value) publishLocal else publishLocalBin)).all(scriptedProjects)
```

Comment:
Not sure if this should vary in CI.  Also it makes the name `publishLocalBinAll` a bit misleading...